modules/deploy/25-couchdb-deploy.erb

```diff
@@ -0,0 +1,114 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: couchdb
+  labels:
+    app: couchdb
+    project: gpii
+    environment: production
+spec:
+  serviceName: couchdb
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: couchdb
+        project: gpii
+        environment: production
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: couchdb
+        image: gpii/couchdb:latest
+        env:
+        - name: COUCHDB_USER
+          value: <%= @devsecrets["couchdb_dev_user"] %>
+          # 
+          # The username should come from a Secret variable
```

Comment:
If you want to use Secrets for this, you could do it now -- you can do the same `@secrets["couchdb_user"]` injection trick in a new Secret manifest.

But I think maybe we can avoid Secrets, use env vars as you're already doing, handle the logic to look up non-dev creds in rake in `:setup_secrets`, and inject those values as we're already doing.